main.tf

```diff
@@ -66,6 +77,17 @@ resource "aws_security_group_rule" "egress" {
   security_group_id = join("", aws_security_group.default[*].id)
 }
 
+resource "aws_security_group_rule" "egress_ipv6" {
+  count             = local.enabled && var.egress_enabled ? 1 : 0
+  description       = "Allow outbound ipv6 traffic"
+  type              = "egress"
+  from_port         = 0
+  to_port           = 0
+  protocol          = "-1"
+  cidr_blocks       = ["::/0"]
```

Comment:
Tests for this seem to fail:
```
TestExamplesComplete 2024-09-06T18:00:04Z retry.go:99: Returning due to fatal error: FatalError{Underlying: error while running command: exit status 1; ╷
│ Error: "::/0" is not a valid IPv4 CIDR block
│ 
│   with module.rds_cluster.aws_security_group_rule.egress_ipv6,
│   on ../../main.tf line 87, in resource "aws_security_group_rule" "egress_ipv6":
│   87:   cidr_blocks       = ["::/0"]
│ 
╵}
```

Reply:
@Benbentwo can you please rerun and check.